src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -537,11 +541,11 @@ protected function getMigrationClass(string $migrationName): string
     public function getMigrationFiles($paths)
     {
         return (new Collection($paths))
-            ->flatMap(fn ($path) => str_ends_with($path, '.php') ? [$path] : $this->files->glob($path.'/*_*.php'))
+            ->flatMap(fn($path) => str_ends_with($path, '.php') ? [$path] : $this->files->glob($path . '/*_*.php'))
             ->filter()
             ->values()
-            ->keyBy(fn ($file) => $this->getMigrationName($file))
-            ->sortBy(fn ($file, $key) => $key)
+            ->keyBy(fn($file) => $this->getMigrationName($file))
```

Comment:
You might be able to replace it with this:
```suggestion
            ->keyBy($this->getMigrationName(...))
```